README.md

```diff
@@ -139,10 +139,10 @@ Install and upgrade:
 
 ### Arch Linux
 
-Arch Linux users can install from the AUR: https://aur.archlinux.org/packages/github-cli/
+Arch Linux users can install from the community repo: https://www.archlinux.org/packages/community/x86_64/github-cli/
 
 ```bash
-yay -S github-cli
```

Comment:
What was wrong with `yay`?

Reply:
`yay` is just a [`pacman`](https://wiki.archlinux.org/index.php/pacman) wrapper with AUR support.
It's not installed by default, and is mostly used as an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers)